src/components/announcement/AnnouncementPathSettingsForm.tsx

```diff
@@ -75,24 +90,58 @@ const AnnouncementPathSettingsForm = ({ announcement, onSave, saveLoading }: Ann
   )
 }
 
-const PathInput: React.FC<{ value?: string[]; onChange?: (value: string[]) => void }> = ({ value, onChange }) => {
+const PathInput: React.FC<{ value?: string[]; onChange?: (value: string[]) => void }> = ({ value = [], onChange }) => {
   const { formatMessage } = useIntl()
+  const [radioValues, setRadioValues] = useState<string[]>(
```

Comment:
The name `radioValue` seems like a single value. I think renaming it to `pathMatchTypes` would be better.